src/Spryker/Install/Executable/CommandLine/CommandLineExecutable.php

```diff
@@ -16,6 +16,7 @@
 
 class CommandLineExecutable implements ExecutableInterface
 {
+    const DEFAULT_TIMEOUT_IN_SECONDS = 600;
     /**
```

Comment:
newline missing?

Reply:
Why does the sniffer not find this?

Reply:
Looks like our sniffer ignores this kind errors, @dereuromark can you confirm and ideally provide a fix for the broken/missing sniff?